Woof/Woof/Model/Sitter/Sitter+DummyData.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+extension Sitter {
+    static let dummySitter = Sitter(
+        id: UUID(),
+        name: "Emily",
+        surname: "Doe",
+        phone: "1234567890",
```

Comment:
Should it start with "+"? Do we have such restriction?

Reply:
We haven't specified this restriction yet.